runtime/interpreter/statictype.go

```diff
@@ -98,6 +99,16 @@ func (t CompositeStaticType) String() string {
 	return string(t.TypeID)
 }
 
+func (t CompositeStaticType) MeteredString(memoryGauge common.MemoryGauge) string {
+	if t.Location == nil {
+		common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(len(t.QualifiedIdentifier)))
+	} else {
+		common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(len(t.TypeID)))
+	}
```

Comment:
Maybe factor out the common parts, the calls to construct the usage and the use of the usage:

```suggestion
	var amount int
	if t.Location == nil {
		amount = len(t.QualifiedIdentifier)
	} else {
		amount = len(t.TypeID)
	}
	common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(amount))
```

Reply:
Added in 49b9ebf